### ISSUE TITLE
Unable to remove manually-added cookies

### ISSUE THREAD

**Issue by espectalll** [stoically/temporary-containers#139]
- Temporary Containers Version: 0.89
- Firefox Version: 59.0.2

### Expected behavior
Clicking on the remove button for a manually-added cookie should delete such cookie.

### Actual behavior
There is no action event for removing a cookie, so clicking does nothing.

### Steps to reproduce
1. Go to the add-on preferences > Advanced > Set Cookies.
2. Add a cookie.
3. Click on the trash can icon next to the new entry.

**Comment by stoically** [comment]
Workaround until fix is published: Open Developer Tools on the Options/Preferences Page (Ctrl+Shift+I) and type

```js
delete preferences.cookies.domain[encodeURIComponent('example.com')]; updateSetCookiesDomainRules();
```

into the console. This will delete all cookies defined for a Domain pattern, `example.com` in this case.

**Comment by stoically** [comment]
Published fix with version 0.90
